recipe/meta.yaml

```diff
@@ -1,43 +1,55 @@
-{% set version = "3.1.0rc0" %}
+{% set version = "3.2.0rc0" %}
 
 package:
   name: iris
   version: {{ version }}
 
 source:
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: d4cf6b827aa15fa6f93078143cd865111e12600b7c68ebfe84e4022c7f3dbfe8
+  sha256: 03589b29d8e20510614ba5d8cf57ce1fca4cf2b128033180beee7e6bf9def202
 
 build:
+  noarch: python
   number: 0
-  skip: true  # [py<37]
 
-requirements:
-  host:
-    - python
-    - pip
-  run:
-    - python
-    - cartopy >=0.18
-    - cf-units
-    - cftime >=1.5.0
-    - dask >=2
-    - matplotlib-base
-    - netcdf4 >=1.4
-    - numpy >=1.14,!=1.15.2
-    - scipy
-    - python-xxhash
+outputs:
+  - name: iris
+    script: build_base.sh
```

Comment:
@lbdreyer Are you certain that `build_base.sh` is platform agnostic? I'm guessing not, particularly for `win`.

We could have:
```bash
  - name: iris
    script: bld.bat  # [win]
    script: build.sh  # [not win]
```

Reply:
very good catch @bjlittle - am surprised the package builder test didn't catch this for a `noarch` output type (sorry, am crashing your party :grin: )

Reply:
Infact, the `build_base.sh` is just a one-liner now that we've established that there are now no `*.npz` files, so I think we can nuke all the scripts and simple do:
```bash
script: python -m pip install --no-deps --ignore-installed .
```
right?